templater/jobs/presubmit/eks-anywhere-build-tooling/autoscaler-1-X-presubmits.yaml

```diff
@@ -0,0 +1,20 @@
+jobName: autoscaler-{{ .releaseBranch }}-presubmit
```

Comment:
Not on you probably but this is not the right base image for this project. Let's also add the eksd release file to this list